src/redshift.c

```diff
@@ -685,6 +686,7 @@ run_continual_mode(const location_provider_t *provider,
 		if (verbose && disabled != prev_disabled) {
```

Comment:
This conditional is wrong, see https://gitlab.com/chinstrap/gammastep/-/issues/2